flake8_strict.py

```diff
@@ -171,7 +175,7 @@ def _process_atom(atom):
 
     # Enforcing trailing commas in list/dict/set comprehensions seems too strict
     # so we won't do it for now even if it is syntactically allowed.
-    has_comprehension_inside = 'comp_for' in {
+    has_comprehension_inside = {'comp_for', 'old_comp_for'} & {
```

Comment:
`has_comprehension_inside = not {'comp_for', 'old_comp_for'}.isdisjoint(...)`?
Wondering if this makes more sense, also means the `has_comprehension_inside` is a bool rather than a set.

Reply:
Sure, I'm happy to change to that.